### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by macklinu**
Hello! I stumbled upon your website and this app and wanted to check it out. When I pulled down the repo, the app would not compile, so I updated various dependencies (Kotlin, Android) and got the app building.

Each commit has more specific details and would probably be easier to review individually.

This seems like a useful development app - looking forward to playing around with it and sharing it with my Android dev coworkers!

### CODE REVIEW

File: app/src/main/kotlin/com/zachklipp/intentsendertool/LaunchAction.kt [hunk 1]
```diff
@@ -43,12 +43,16 @@ public enum class LaunchAction(val name: String) {
...
   public abstract fun resolve(intent: Intent, packageManager: PackageManager): IntentTargets
   public abstract fun launch(intent: Intent, context: Context)
 
   override fun toString(): String {
-    return name
+    return launchName
+  }
+
+  companion object {
+    public fun getLaunchTypeNames(): List<String> = LaunchAction.values.map { it.launchName }
```
Comment (zach-klippenstein) [inline app/src/main/kotlin/com/zachklipp/intentsendertool/LaunchAction.kt]: I don't think this ever used.

File: app/src/main/kotlin/com/zachklipp/intentsendertool/MainActivity.kt [hunk 2]
```diff
@@ -29,8 +28,8 @@ public class MainActivity : Activity() {
   private val mResolveView: IntentTargetView by bindView(R.id.resolve_view)
   private val mNoResultsText: TextView by bindView(R.id.no_results_text)
 
-  private val mLaunchAdapter: ArrayAdapter<LaunchAction> by Delegates.lazy() {
-    ArrayAdapter(this, R.layout.support_simple_spinner_dropdown_item, LaunchAction.values())
+  private val mLaunchAdapter: ArrayAdapter<LaunchAction> by lazy {
+    ArrayAdapter(this, R.layout.support_simple_spinner_dropdown_item, LaunchAction.values)
```
Comment (zach-klippenstein) [inline app/src/main/kotlin/com/zachklipp/intentsendertool/MainActivity.kt]: `.values` should be a method call: http://blog.jetbrains.com/kotlin/2015/12/kotlin-1-0-beta-3-is-out/#more-3263

**Comment by zach-klippenstein** [comment]
This is awesome, thanks! Just a couple comments about enum values, but otherwise LGTM.

File: build.gradle [hunk 3]
```diff
@@ -1,29 +1,10 @@
...
-}
-
 ext {
-    androidSupportVersion = '21.0.0'
-    appCompatVersion = '21.0.+'
-    kotlinVersion = '0.10.195'
-    kotterknifeVersion = '0.1.0-SNAPSHOT'
+    deps = [
+        androidSupport: '23.0.1',
+        appCompat: '23.0.1',
```
Comment (zach-klippenstein) [inline build.gradle]: 23.1.1

**Comment by macklinu** [comment]
Thanks for taking a look. Addressed the PR comments - let me know if I can squash into one commit or anything else before this can be merged.

**Comment by zach-klippenstein** [comment]
Yea, go ahead and squash and I'll merge it.

**Comment by macklinu** [comment]
Squashed :+1:

**Pull request merged**